job-descriptions/engineering/engineering-manager-campaigns.md

```diff
@@ -0,0 +1,47 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Engineering Manager - Campaigns
+
+We are looking for an experienced engineering leader to manage our cross-functional engineering team building [campaigns](https://about.sourcegraph.com/handbook/engineering/campaigns). You will partner with our [product team](https://about.sourcegraph.com/handbook/product) to define team goals, and develop a roadmap to achieve those goals. Our product and team is very technical; we expect you to have a strong background as an engineer so that you can contribute to technical discussions and contribute code when necessary/appropriate. The team is small today but there is a lot more work to be done and we will expect you to grow the team in a sustainable way that maintains our high standards.
```

Comment:
I'm not sure what the goal of this paragraph is, but the "You will partner with [...]" sentence sounds like it's a description of tasks for the manager.

Should we also mention "improve existing and establish new processes", "lead the team through the monthly iterations by helping in the planning process", etc.?